internal/pkg/deploy/pipeline_test.go

```diff
@@ -203,6 +204,17 @@ func TestPipelineBuild_Init(t *testing.T) {
 		defaultImage   = "aws/codebuild/amazonlinux2-x86_64-standard:3.0"
 		defaultEnvType = "LINUX_CONTAINER"
 	)
+	yamlNode := yaml.Node{}
+	policyDocument := []byte(`
+  Statement:
+    Action: '*'
+    Effect: Allow
+    Resource: '*'
+  Version: 2012-10-17`)
+
+	if err := yaml.Unmarshal(policyDocument, &yamlNode); err != nil {
+		t.Errorf("unmarshal error:  %v", err)
```

Comment:
let's do `require.NoError(t, err)`, just to be consistent with the other failures

Reply:
I think we need `require.Error(t, err)` as we are going inside if only when error occurs. What do you think about it?

Reply:
i think you can remove the if check and just do something like this:
```
require.NoError(t, yaml.Unmarshal(policyDocument, &yamlNode))
```